mapdata/WarcraftLegacies/AbilityData/Skin/875770177.json

```diff
@@ -20,7 +20,7 @@
       "Level": 1,
       "Id": 828536161,
       "Type": 3,
-      "Value": "Heals a target friendly unit for 800 mana over 20 seconds. Has a 60 seconds cooldown."
+      "Value": "Heals a target friendly unit for 600 mana over 20 seconds. Has a 60 seconds cooldown."
```

Comment:
While you're at it could you modify this to generate the tooltip from object data?